doc/source/whatsnew/v2.2.0.rst

```diff
@@ -858,7 +858,7 @@ MultiIndex
 ^^^^^^^^^^
 - Bug in :meth:`MultiIndex.get_indexer` not raising ``ValueError`` when ``method`` provided and index is non-monotonic (:issue:`53452`)
 
-I/O
+IO
```

Comment:
Decided to drop slash here, as "IO" has been already added to exception list

Reply:
I think this should be left as is.